### PR TITLE
Add TF vision model code samples

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Add:

- TF_VISION_BASE_MODEL_SAMPLE
- TF_VISION_SEQ_CLASS_SAMPLE

I tested them manually (not through doctest).

(I will add code samples for TF speech models in another PR)

### CODE REVIEW

**Review by gante — APPROVED** [review]
👍

**Review by sgugger — APPROVED** [review]
LGTM! Thanks for adding those!

File: src/transformers/utils/doc.py [hunk 1]
```diff
@@ -794,6 +794,52 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
...
+    >>> inputs = feature_extractor(image, return_tensors="tf")
+    >>> outputs = model(**inputs)
+
+    >>> last_hidden_states = outputs.last_hidden_state
+    >>> list(last_hidden_states.shape)
+    {expected_output}
+    ```
+"""
+
+TF_VISION_SEQ_CLASS_SAMPLE = r"""
```
Comment (NielsRogge) [inline src/transformers/utils/doc.py:819]:
```suggestion
TF_IMAGE_CLASSIFICATION_SAMPLE = r"""
```

File: src/transformers/utils/doc.py [hunk 2]
```diff
@@ -802,6 +848,8 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     "MaskedLM": TF_MASKED_LM_SAMPLE,
     "LMHead": TF_CAUSAL_LM_SAMPLE,
     "BaseModel": TF_BASE_MODEL_SAMPLE,
+    "VisionBaseModel": TF_VISION_BASE_MODEL_SAMPLE,
+    "ImageClassification": TF_VISION_SEQ_CLASS_SAMPLE,
```
Comment (NielsRogge) [inline src/transformers/utils/doc.py:852]:
```suggestion
    "VisionBaseModel": TF_VISION_BASE_MODEL_SAMPLE,
    "ImageClassification": TF_IMAGE_CLASSIFICATION_SAMPLE,
```

**Pull request merged**